include/anyrpc/server.h

```diff
@@ -22,24 +22,9 @@
 #define ANYRPC_SERVER_H_
 
 #if defined(ANYRPC_THREADING)
-# if defined(__MINGW32__)
-// These constants are not defined for mingw but are needed in the following libraries
-#  ifndef EOWNERDEAD
-#   define EOWNERDEAD       133    /* File too big */
-#  endif
-#  ifndef EPROTO
-#   define EPROTO    134    /* Protocol error */
-#  endif
-
-#  include "internal/mingw.thread.h"
-#  include <mutex>
-#  include "internal/mingw.mutex.h"
-#  include "internal/mingw.condition_variable.h"
```

Comment:
These internal files are no longer used, I guess they can be removed from the source tree?

Reply:
Ping @cwassmuth